client/src/curriculum/modules-list.scss

```diff
@@ -0,0 +1,236 @@
+@use "../ui/vars" as *;
+
+.curriculum-content-container.with-sidebar,
+.curriculum-content-container {
+  .curriculum-content {
+    .modules {
+      input[type="radio"]:not(:checked) ~ a,
+      input[type="radio"]:not(:checked) ~ ol {
+        display: none;
+      }
+    }
+
+    .module-contents {
+      h2 {
+        width: 100%;
+      }
+    }
+
+    ol.modules-list-list {
+      display: grid;
+      grid-template-areas:
+        "a   b   c"
+        "hr  hr  hr"
+        "mod mod mod"
+        "t   t   t";
```

Comment:
Not all of the area names are self-explanatory:

- What do we use `a`/`b`/`c` for?
- And what does `t` stand for?